modules/k9s.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs, config, lib, ... }:
+
+let
+  skinFile = config.lib.stylix.colors {
+    templateRepo = pkgs.fetchFromGitHub {
+      owner = "krgn";
+      repo = "base16-k9s";
+      rev = "930124928781551c696a8a08695e6e577adff441";
+      sha256 = "Bs/RFm6jRPMNQH36iqlLdgqj0nzy34mUAWP67cBnWug=";
+    };
+  };
+
+in {
+  options.stylix.targets.k9s.enable =
+    config.lib.stylix.mkEnableTarget "k9s" true;
+
+  config = lib.mkIf config.stylix.targets.k9s.enable {
+
+    home-manager.sharedModules = [{
+      # Make sure there's no conflicting config
+      programs.k9s.skin = null;
```

Comment:
Would it be feasible to use this option to set the scheme?

That would avoid creating the config file when k9s is not installed.

Reply:
I could try using `yq` to convert the yaml to json and then call `builtins.fromJSON`. I'll have a go at it later.

Reply:
Took me a few attempts, but it works now :smile: